### DISCUSSION TITLE
Was bedeutet "Kartenerstellung aktivieren"

### DISCUSSION THREAD

**Discussion by SamLowrie** [copystring/ioBroker.roborock#79] [Q&A] [answered]
Mir ist nicht ganz klar was der Punkt "Kartenerstellung aktivieren" bedeutet. Hat der Punkt Einfluß auf die Karte des Roborocks selbst (und es besteht die Gefahr das was mit der Karte passiert)? Oder ist das rein passiv eine Aktualisierung der Karte im Adapter?

**Comment by copystring** [comment, marked as answer]
Das ist rein für das erstellen der Karte für den Adapter. Die Option existiert deshalb weil sie nicht auf jedem Gerät funktioniert. z.B. ein Raspberry Pi 4

**Reply by hallerdavid** [reply]
Warum funktioniert diese Funktion nicht auf einem Raspberry Pi 4? Bei mir wird die Karte über den Webserver sauber angezeigt?

**Reply by copystring** [reply]
Auf dem Pi läuft das nicht, weil die canvas bibliothek fehlerhaft ist.

**Reply by hallerdavid** [reply]
Okay... aber nochmal: ich hab ne Karte! 🤔 auf nem pi! 🤭

**Reply by copystring** [reply]
Japp, das war bei mir auch so. Mehrere Nutzer meldeten sich, dass die Karte nicht geht. Das war immer auf einem Pi. Ich wollte es reproduzieren und es lief sofort... 🤷‍♂️

**Reply by hallerdavid** [reply]
Ohje. Dann hab ich wohl Glück :)

**Reply by copystring** [reply]
Es gibt sicherlich einen Grund dafür, dass es bei anderen nicht läuft. Das wird wohl aber an der Installation lokal auf dem Pi liegen und nicht am Adapter, da es ja bei dir und mir läuft. Aber ja, da hast du Glück gehabt.

**Comment by SamLowrie** [comment]
Danke für die Antwort, dann kann ich ja beruhigt weiter testen! :-)
